src/test/java/spoon/test/refactoring/testclasses/TryRename.java

```diff
@@ -0,0 +1,16 @@
+package spoon.test.refactoring.testclasses;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.LOCAL_VARIABLE, ElementType.PARAMETER, ElementType.FIELD})
+public @interface TryRename {

```

Comment:
I really like the idea of having an annotation for testing, this is beautifully declarative.

may we should have `Test` in its name for more clarity.